src/Core/Settings.cpp

```diff
@@ -16,9 +16,12 @@ namespace ErrorCodes
     extern const int UNKNOWN_ELEMENT_IN_CONFIG;
 }
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wframe-larger-than="
 
 IMPLEMENT_SETTINGS_TRAITS(SettingsTraits, LIST_OF_SETTINGS)
```

Comment:
@alexey-milovidov latest clang warning
```
Documents/ClickhouseBrowse/src/Core/Settings.cpp:20:1: error: stack frame size of 81128 bytes in lambda expression [-Werror,-Wframe-larger-than=]
IMPLEMENT_SETTINGS_TRAITS(SettingsTraits, LIST_OF_SETTINGS)
^
Documents/ClickhouseBrowse/src/Core/BaseSettings.h:817:46: note: expanded from macro 'IMPLEMENT_SETTINGS_TRAITS'
        static const Accessor the_instance = [] \
```
```
Documents/ClickhouseBrowse/build$ clang++-11 --version
Ubuntu clang version 11.0.1-++20201103062930+ef4ffcafbb2-1~exp1~20201103053545.129
```

Reply:
Ok. Frame size limit is currently just a precaution.